enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -214,4 +259,24 @@ public RoleResult( String role, Set<String> users )
             this.users.addAll( users );
         }
     }
+
+    public class TransactionResult
+    {
+        public final String username;
+        public final String transaction;
+
+        public TransactionResult( KernelTransaction tx )
+        {
+            if ( tx.mode() instanceof AuthSubject )
+            {
+                AuthSubject authSubject = (AuthSubject) tx.mode();
+                username = authSubject.name();
+            }
+            else
+            {
+                username = "-";
+            }
+            transaction = tx.toString();
```

Comment:
So what we really wanted to report was a transaction id, but we didn't find any way of extracting that. Is there such a thing? This would be very useful if we at some point want to kill specific transactions as opposed to all of them, but that might not be relevant, I don't know. Otherwise I agree that report simply the numbers of transactions is the best choice.